src/components/ContentWrap/ContentWrap.js

```diff
@@ -2,14 +2,83 @@
  * This function returns a div that contains the left and right columns for issuer
  * @returns An empty element with the left and right column inside.
  */
+import { useEffect, useState } from 'react'
 import ColumnLeftWrap from './../ColumnLeft/ColumnLeftWrap'
 import ColumnRightWrap from './../ColumnRight/ColumnRightWrap'
+import useGetLoopedPresentProofRecords from '../../interface/hooks/use-get-looped-present-proof-records'
 
 export default function ContentWrap({ origin }) {
+  const [records, setRecords] = useState(null)
+  const [dataConnRecordEvents, setDataConnRecordEvents] = useState([])
+  const [statusRecordEvents, errorRecordEvents, startGetRecordsHandler] =
+    useGetLoopedPresentProofRecords()
+
+  useEffect(() => {
+    const setDataFn = (resRecordsData) => {
+      setRecords(resRecordsData)
+      setDataConnRecordEvents((prevData) => {
+        prevData = [...prevData]
+        resRecordsData.forEach((record) => {
+          const {
+            connection_id: connId,
+            pres_ex_id: exId,
+            updated_at: updatedAt,
+          } = record
+
+          let connIdIndex = prevData.findIndex((c) => connId in c)
+          if (connIdIndex === -1) {
+            connIdIndex = prevData.push({ [connId]: [] }) - 1
```

Comment:
this is weird. why can't we just say null? To be fair the whole lot looks a bit weird (hard to understand)

Reply:
reverted ContentWrap change